products/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM eclipse-temurin:17-jdk-alpine
```

Comment:
hier werden quasi alle dependencies festgelegt die der Docker für unser java programm braucht oder ?

Reply:
Da definiert man die Grundlage für den Container, in dem Fall also java 17